client-tez/src/main/java/org/apache/tez/dag/app/RssDAGAppMaster.java

```diff
@@ -336,6 +351,16 @@ public static void main(String[] args) {
         }
       }
 
+      if (conf.getBoolean(
```

Comment:
Could  we add  a private field in the `RssDAGAppMaster` instead of using this config option repeatly?

Reply:
I move this to serviceInit, seems there are no need to add a private field.

Reply:
I got it. This is a static method.

Reply:
OK, then can I mark this resolved?